Cargo.toml

```diff
@@ -54,6 +54,7 @@ tokio-core = "0.1.17"
 toml = "0.4"
```

Comment:
The one who read this commit message will not know what is merged.
How about "Implement voting application"?

Reply:
Is the "Merge Voting module" commit the same as the previous PRs?
If you did not change any code, I'll not check the code in detail.

Please tell me if you changed something.

Reply:
@majecty it is the same as previous PRs.